rust/src/lib/ip.rs

```diff
@@ -558,6 +565,20 @@ impl InterfaceIpv6 {
                 addr.mptcp_flags = None;
             }
         }
+        if let Some(token) = self.token.as_mut() {
+            if self.autoconf == Some(false)
+                && !(token.is_empty() || token == "::")
+            {
+                return Err(NmstateError::new(
+                    ErrorKind::InvalidArgument,
+                    format!(
+                        "Desired IPv6 token '{token}' cannot \
+                        be applied with IPv6 autoconf disabled"
```

Comment:
Just a nit, should we suggest the user to configure the IPv6 token to "" or "::"? What do you think?